server/src/test/java/org/apache/gravitino/server/web/TestConfigServlet.java

```diff
@@ -43,4 +45,23 @@ public void testConfigServlet() throws Exception {
             "{\"gravitino.authorization.enable\":false,\"gravitino.authenticators\":[\"simple\"]}");
     configServlet.destroy();
   }
+
+  @Test
+  public void testConfigServletWithVisibleConfigs() throws Exception {
+    ServerConfig serverConfig = new ServerConfig();
+    serverConfig.set(
+        Configs.VISIBLE_CONFIGS,
+        Lists.newArrayList(Configs.AUDIT_LOG_FORMATTER_CLASS_NAME.getKey()));
+    serverConfig.set(Configs.AUDIT_LOG_FORMATTER_CLASS_NAME, "test");
```

Comment:
Em ... this is an interesting test case. Can we use something more realistic? I'm not sure who wants to know the audit log formatter class? Maybe there are something I missed.

Reply:
OK, I can use a more realistic case.